monkey-runner/build.gradle

```diff
@@ -0,0 +1,24 @@
+task wrapper(type: Wrapper) {
+    gradleVersion = versions.gradleWrapperVersion
+}
+
+buildscript {
+    apply from: 'dependencies.gradle'
+
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath gradlePlugins.gradle
+        classpath gradlePlugins.gradleAndroidCommandPlugin
+        classpath gradlePlugins.bintrayRelease
+    }
+}
+
+apply from: 'dependencies.gradle'
```

Comment:
Sorry for the delay :o - fixed here 85fe33d.

As the task wrapper was using a version defined in dependencies.gradle, restricting it to subprojects meant that the task wrapper failed, but instead of having a separate file to define the wrapper version, I opted for the duplication in `/build.gradle` and `/sample/build.gradle` with regards to the declaration of the wrapper version.